README.md

```diff
@@ -138,63 +131,163 @@ In support of the various browser privacy models, first-party sets only control
+exampleA.co.uk<br>
+<br>
+exampleA.ca</td>
+    </tr>
+    <tr>
+      <td><br>
+common eTLD (effective Top Level Domain)</td>
+      <td><br>
+Reserved for domains that share a common eTLD as the set primary. These are not <a href="https://www.iana.org/domains/root/db">IANA-managed TLDs</a>, but domains added to the <a href="https://publicsuffix.org/">PSL</a> for improved security isolation.<br>
+<br>
+<br>
+<em>Requires common ownership.</em></td>
+      <td><br>
+exampleA.gov.uk<br>
+<br>
+exampleB.gov.uk</td>
+    </tr>
+    <tr>
+      <td><br>
+service</td>
+      <td><br>
+Reserved for utility or sandbox domains.<br>
+<br>
+<br>
+<em>Requires common ownership.</em></td>
+      <td><br>
+exampleA-usercontent.com<br>
+<br>
+exampleA-cdn.net</td>
+    </tr>
+    <tr>
+      <td><br>
+associated</td>
+      <td><br>
+Reserved for domains whose affiliation with the set primary is clearly presented to users (e.g., an About page, header or footer, shared branding or logo, or similar forms).</td>
+      <td><br>
+exampleA-affiliated.com<br>
+<br>
+exampleB.com*<br>
+<br>
+exampleC.com*<br>
+<br>
+<br>
+*where exampleB and exampleC are separately owned websites, but clearly present their affiliation with exampleA to users</td>
+    </tr>
+  </tbody>
+</table>
+
+While we think this subset framework has the clear benefit of furthering transparency around why a domain has been added to a set, the primary value to this framework is that the browser could handle each subset differently:
+
+<table>
+  <thead>
+    <tr>
+      <th><strong>Subset type</strong></th>
+      <th><strong>Subset definition</strong></th>
+      <th><strong>Example browser handling policy</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>ccTLD (country code Top Level Domain)</td>
+      <td>Reserved for variations for a particular country or a geographical area. <br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>common eTLD (effective Top Level Domain)</td>
+      <td>Reserved for domains that share a common eTLD as the set primary. These are not <a href="https://www.iana.org/domains/root/db">IANA-managed TLDs</a>, but domains added to the <a href="https://publicsuffix.org/">PSL</a> for improved security isolation.<br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>service</td>
+      <td>Reserved for utility or sandbox domains.<br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access. Not allowed to be the top-level domain in a storage access grant.</td>
+    </tr>
+    <tr>
+      <td>associated</td>
+      <td>Reserved for domains whose affiliation with the set primary is clearly presented to users (e.g., an About page, header or footer, shared branding or logo, or similar forms).</td>
+      <td>Limit of 3* domains. If greater than 3, auto-reject access.<br>
```

Comment:
An additional rule might make this practical: a pull request that adds an "associated" set must include user research to substantiate that the set is clearly presented to users.

 * research should cover a representative sample of the audience for the sites that are members of the proposed set
 * research should document the understanding on the part of the users of the relationship among site members
 * results should be statistically significant 
 * researchers should be available to answer questions on the PR.

Reply:
Thanks for the suggestion, Don! A few personal thoughts from my side:

I agree with your concern about long discussions/pile-ons with subjective opinions of what constitutes "clearly presented" and we should try to avoid that.

However, I feel like asking for dedicated user research is adding significant cost to submission. It's a barrier to smaller sites and sites from non-English speaking countries, and even mid-sized organizations may struggle coordinating this kind of project.

Conversely, if an attacker falsely claims two domains have a clear affiliation, faking or manipulating user research in order to pass doesn't seem unthinkable. So we're setting up the enforcement entity/community to the challenge of disproving the presented results, which they would always strive to do if the association is not subjectively clear to them. As such, "clear presentation" seems like it would end up to be the main criterion either way?

In the end, I'm not sure that user research would actually sway most people's subjective opinion about whether two domains are clearly affiliated.

Given that, it seems a bit excessive to ask of developers by default, though I think it's interesting to consider as an optional step.

Reply:
A problem with having the public reviewers directly review the set is that they're not necessarily going to get the same experience as the real users who are using the set member sites. A set could do some pages that are "clearly presented" (home page, privacy policy) for public review, and then promote other pages that are not clearly presented or easily findable. (For example, a clinic site and an online pharmacy could make their home pages clearly common set members, then use social media ads to promote "independent doctor recommendations" and "product info" pages that are designed to appear independent).

FPS is an exchange of value for value -- the set members get additional information from users, and the users get additional checks on the practices and reputations of the set members. Set members are asking for something from users, so should offer something in exchange -- research that reflects people's real experience with the set members. And the research does not have to be a separate project -- it can be an expansion of the user studies that set members are already doing.

There is a risk that a user researcher could present a completely bogus study, but each researcher would only be able to get away with that once. New and unknown researchers would receive additional attention.

Reply:
I think https://github.com/WICG/first-party-sets/issues/101 is somewhat picking up on this topic, so maybe we can continue to discuss over there.